scripts/ci/osti-nmc/run.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+_CMAKE_DIR="$HOME/cmake/${CI_ENVIRONMENT}"
+export PATH="$_CMAKE_DIR/bin:$PATH"
+export ACLOCAL_PATH="$_CMAKE_DIR/share/aclocal:$ACLOCAL_PATH"
+
+# Parse the branch name used by the PR
+API_BASE="https://api.github.com/repos/ornladios/adios2"
+REALBRANCH="${CI_COMMIT_REF_NAME}"
+
+export CI_SITE_NAME="OSTI Gitlab (NMC)"
+
+CLUSTER_NAME="$SLURM_CLUSTER_NAME"
+if [ -z "$CLUSTER_NAME" ] ; then
+    CLUSTER_NAME='unknown'
+fi
+
+export CI_BUILD_NAME="${REALBRANCH}_${CLUSTER_NAME}_${CI_ENVIRONMENT}"
+export CI_COMMIT_REF=${CI_COMMIT_SHA}
+export CI_SOURCE_DIR="${PWD}"
+export CI_ROOT_DIR="$( dirname "${CI_SOURCE_DIR}" )"
+export CI_BIN_DIR="${CI_ROOT_DIR}/build/${CI_PIPELINE_ID}"
```

Comment:
A little nitpicky but let's try to keep this at the same level as the source, i.e. `export CI_BIN_DIR="${CI_ROOT_DIR}/build_${CI_PIPELINE_ID}"`.